### PR TITLE
Mdesforges/solitaire

### PR DESCRIPTION
**Pull request by Maxime172000**

### CODE REVIEW

File: apps/deepDive/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "lint": "eslint .",
-    "dev": "react-native start",
+    "devaaa": "react-native start",
```
Comment (klanthier) [inline apps/deepDive/package.json:9]: ?

File: apps/solitaire/package.json [hunk 2]
```diff
@@ -0,0 +1,209 @@
+{
+  "name": "solitaireplusreward",
+  "version": "1.1.0",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "lint": "eslint .",
+    "dev": "react-native start --reset-cache",
```
Comment (klanthier) [inline apps/solitaire/package.json:9]: Can you add the reset cache flags to the other games as well

File: packages/wrapper/src/modules/graphql/queries/challenge.gql [hunk 3]
```diff
@@ -6,8 +6,8 @@ query getChallenge($id: UUID!) {
   }
 }
 
-mutation startChallenge($cycleId: UUID, $metadata: String) {
-  startChallenge(cycleId: $cycleId, metadata: $metadata, gameId: "deepdive") {
+mutation startChallenge($cycleId: UUID, $metadata: String, $gameId: String!) {
```
Comment (klanthier) [inline packages/wrapper/src/modules/graphql/queries/challenge.gql:9]: You changed this value, but I don't see a change in the wrapper to use a value instead ?

**Review by klanthier — CHANGES_REQUESTED** [review]

File: packages/wrapper/src/modules/graphql/queries/challenge.gql (hunk 3 above)
Comment (Maxime172000) [reply packages/wrapper/src/modules/graphql/queries/challenge.gql:9]: good catch, i did fix the generate script to run on post install, so no need to worry about it again

**Review by klanthier — APPROVED** [review]
lgtm

**Pull request merged**